### PR TITLE
incorporate updates to from styles repo 

### PR DESCRIPTION
**Pull request by raynamharris**
I incorporated all the updates from the styles repos from June  2018 to April 2019. There were about 10 merge conflicts where Spanish and English text conflicted, so I fixed all the conflicts by keeping the English text. 

cc @fmichonneau

### CODE REVIEW

**Comment by raynamharris** [comment]
Hmm. I tried to build the style-es site locally, and I got the following error

```
  Liquid Exception: Could not find document 'aio.md' in tag 'link'. Make sure the document exists and the path is correct. in CODE_OF_CONDUCT.md
jekyll 3.8.5 | Error:  Could not find document 'aio.md' in tag 'link'.

Make sure the document exists and the path is correct.
```

I'm not sure what this means or how to debug it

**Comment by rgaiacs** [comment]
@raynamharris The `aio.md` file is created by `bin/lesson_initialize.py`. You can copy https://github.com/carpentries/styles-es/blob/gh-pages/bin/boilerplate/aio.md safely.

File: LICENSE.md [hunk 1]
```diff
@@ -5,10 +5,18 @@ root: .
 ---
 ### Material didáctico
 
+<<<<<<< HEAD
 Todo el material de instrucción de Software Carpentry and Data Carpentry está
 disponible bajo la licencia [Creative Commons Attribution][cc-por-humano]. El siguiente es un resumen de licencia
 (y no un sustituto) del [texto legal completo de CC BY 4.0
 licencia][cc-by-legal].
+=======
```
Comment (fmichonneau) [inline LICENSE.md]: this merge conflict needs to be addressed

**Review by fmichonneau — COMMENTED** [review]
This looks good! I think there is only one merge conflict left to address.

It's normal that you can't build this locally by itself as it doesn't have all the pieces needed to be a full template. With the current setup, you'd need to run `./bin/lesson_initialize.py` before being able to run `make serve`. However, if you do this, make sure you don't commit the "new" files to the repo. Alternatively, once this is merged, start pulling this new version of the template to one of the existing lesson, and run `make serve` there.

File: LICENSE.md (hunk 1 above)
Comment (raynamharris) [reply LICENSE.md]: thanks for catching that one! fixed with https://github.com/carpentries/styles-es/pull/15/commits/9de85b80a31869d8b7c8dca5227f73e94e8d0643

**Comment by raynamharris** [comment]
thanks for the feedback @fmichonneau and @rgaiacs. 

I fixed the merge conflict. i'll wait to test it our once i merge this into a real lesson. thanks!

**Comment by raynamharris** [comment]
Is this ready to merge? @fmichonneau

**Comment by fmichonneau** [comment]
yes, that looks good!

**Comment by orchid00** [comment]
hi @fmichonneau 
Could you please have a look at this PR again and merge it if possible?

**Comment by fmichonneau** [comment]
replaced by #16

**Pull request closed without merging**